src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+# Application properties
+app.name=DatabaseService
+app.version=1.0.0
+
+# File storage configuration
+database.storage.path=D:\\Files
```

Comment:
Please leave the value of `database.storage.path` empty here. This path is platform and disk name related and should be filled individually.